ee/clickhouse/queries/retention/retention_actors.py

```diff
@@ -57,70 +26,18 @@ def is_aggregating_by_groups(self) -> bool:
-        )
-        return_query, return_params = _get_condition(returning_entity, table="e", prepend="returning")
-        return_query_formatted = f"AND {return_query}"
-
-        reference_date_from = self._filter.date_from
-        reference_date_to = self._filter.date_from + self._filter.period_increment
-        date_from = self._filter.date_from + self._filter.selected_interval * self._filter.period_increment
-        date_to = date_from + self._filter.period_increment
-
-        target_event_query, target_params = RetentionEventsQuery(
+        actor_query = _build_actor_query(
             filter=self._filter,
-            team_id=self._team.pk,
-            event_query_type=RetentionQueryType.TARGET_FIRST_TIME
-            if is_first_time_retention
-            else RetentionQueryType.TARGET,
-        ).get_query()
-
-        actor_field_name = f"{get_aggregation_target_field(cast(Optional[GroupTypeIndex], self._filter.aggregation_group_type_index), self.EVENT_TABLE_ALIAS, self.DISTINCT_ID_TABLE_ALIAS)} AS actor_id"
-
-        target_params.update(
-            {
-                "target_start_date": reference_date_from.strftime(
-                    "%Y-%m-%d{}".format(" %H:%M:%S" if self._filter.period == "Hour" else " 00:00:00")
-                ),
-                "target_end_date": reference_date_to.strftime(
-                    "%Y-%m-%d{}".format(" %H:%M:%S" if self._filter.period == "Hour" else " 00:00:00")
-                ),
-            }
+            team=self._team,
+            filter_by_breakdown=self._filter.breakdown_values or (0,),
+            selected_interval=self._filter.selected_interval,
         )
 
-        return (
-            RETENTION_PEOPLE_SQL.format(
-                target_event_query=target_event_query,
-                returning_query=return_query_formatted,
-                filters=prop_filters,
-                actor_field_name=actor_field_name,
-                person_join=""
-                if self.is_aggregating_by_groups
-                else f"JOIN ({get_team_distinct_ids_query(self._team.pk)}) pdi on e.distinct_id = pdi.distinct_id",
-            ),
-            {
-                "team_id": self._team.pk,
-                "start_date": date_from.strftime(
-                    "%Y-%m-%d{}".format(" %H:%M:%S" if self._filter.period == "Hour" else " 00:00:00")
-                ),
-                "end_date": date_to.strftime(
-                    "%Y-%m-%d{}".format(" %H:%M:%S" if self._filter.period == "Hour" else " 00:00:00")
-                ),
-                "offset": self._filter.offset,
-                **target_params,
-                **return_params,
-                **prop_filter_params,
-            },
-        )
+        return actor_query, {}
 
 
+# Note: This class does not respect the entire flor from ActorBaseQuery because the result shape differs from other actor queries
 class ClickhouseRetentionActorsByPeriod(ActorBaseQuery):
```

Comment:
Just brought over a lot of the functions that were being used to create this functionality